### PR TITLE
Update ChatQnA yaml and set retriever's TEI_EMBEDDING_ENDPOINT

### PR DESCRIPTION
**Pull request by bigclouds**
## Description

Set TEI_EMBEDDING_ENDPOINT to use local embedding service

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Others (enhancement, documentation, validation, etc.)

## Tests

docker compose up -d

### CODE REVIEW

File: ChatQnA/docker_compose/intel/hpu/gaudi/compose.yaml [hunk 1]
```diff
@@ -59,6 +59,7 @@ services:
       https_proxy: ${https_proxy}
       REDIS_URL: ${REDIS_URL}
       INDEX_NAME: ${INDEX_NAME}
+      TEI_EMBEDDING_ENDPOINT: ${TEI_EMBEDDING_ENDPOINT}
```
Comment (lkk12014402) [inline ChatQnA/docker_compose/intel/hpu/gaudi/compose.yaml:62]: hi, the environment variable should be TEI_ENDPOINT? refer this https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L37

File: ChatQnA/docker_compose/intel/cpu/xeon/compose_qdrant.yaml [hunk 2]
```diff
@@ -53,6 +53,7 @@ services:
       QDRANT_HOST: ${QDRANT_HOST}
       QDRANT_PORT: 6333
       INDEX_NAME: ${INDEX_NAME}
+      TEI_EMBEDDING_ENDPOINT: ${TEI_EMBEDDING_ENDPOINT}
```
Comment (lkk12014402) [inline ChatQnA/docker_compose/intel/cpu/xeon/compose_qdrant.yaml:56]: https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/qdrant/langchain/config.py#L17

Comment (bigclouds) [reply ChatQnA/docker_compose/intel/cpu/xeon/compose_qdrant.yaml:56]: https://github.com/bigclouds/GenAIExamples/blob/main/ChatQnA/docker_compose/intel/cpu/xeon/README_qdrant.md#1-build-retriever-image    ; this link tells how  retriever-qdrant:latest image is build.

comps/retrievers/qdrant/haystack/qdrant_config.py is to get variable from env. 
EMBED_ENDPOINT = os.getenv("TEI_EMBEDDING_ENDPOINT", "")

File: ChatQnA/docker_compose/intel/hpu/gaudi/compose.yaml (hunk 1 above)
Comment (bigclouds) [reply ChatQnA/docker_compose/intel/hpu/gaudi/compose.yaml:62]: https://gitee.com/bigclouds99/GenAIExamples/tree/1.1_with_wrapper/ChatQnA/docker_compose/intel/hpu/gaudi#2-build-retriever-image

https://github.com/opea-project/GenAIComps/blob/main/comps/retrievers/redis/langchain/retriever_redis.py#L34

Comment (lkk12014402) [reply ChatQnA/docker_compose/intel/hpu/gaudi/compose.yaml:62]: Sorry, I got the wrong line number

**Review by lkk12014402 — APPROVED** [review]

**Comment by bigclouds** [comment]
@lvliang-intel hi, please take time to review this pr.

**Pull request merged**